pilot/proxy/envoy/mixer.go

```diff
@@ -201,42 +201,48 @@ func mixerHTTPRouteConfig(role proxy.Node, instances []*model.ServiceInstance, c
 					},
 				},
 			},
+			DisableCheckCalls:  outboundRoute || mesh.DisablePolicyChecks,
+			DisableReportCalls: outboundRoute || role.Type == proxy.Ingress,
```

Comment:
why disable report at ingress? Report should have no issue right?

Reply:
Human error. I mixed up the boolean flag usage for building opaque route config. This has been reverted and instead we disable forwarding mixer attributes to the app for sidecar inbound routes.